src/nupic/research/fast_temporal_memory.py

```diff
@@ -175,7 +175,7 @@ def columnForCell(self, cell):
     """
     self._validateCell(cell)
 
-    return int(cell.idx / self.cellsPerColumn)
+    return int(cell / self.cellsPerColumn)
```

Comment:
@chetan51 Well, two reasons:

1. One the way the function was implemented before was at odds with the docstring. The docstring specifies an int parameter, but the implementation expected a Cell object. Since no code that I wasn't already modifying was using the old version, this seemed like an easy way to fix an inconsistency in the API.

2. Previously, code that accessed attributes like `activeCells` in `FastTemporalMemory` directly would get a collection of `Cell` objects, whereas code that accessed the same attribute in `TemporalMemory` would get a collection of indices, so it made sense that the former's implementation of `columnForCell` would accept `Cell`s whereas the latter's would accept indices. After changing the callers to use the accessors, we've standardized on returning indices for both implementations, and so this function can always expect indices.

Though, actually, now that I think of it, since `FastTemporalMemory` inherits from `TemporalMemory` we can delete this definition since two are now identical.